app/components/Controls.tsx

```diff
@@ -70,18 +79,18 @@ export const Controls = ({
               <a
                 href="#"
                 onClick={(e: any) => microphoneToggle(e)}
-                className={`w-20 sm:w-24 py-4 px-2 sm:px-8 rounded-s-full font-bold bg-[#101014] text-light-900 text-sm sm:text-base flex items-center justify-center group`}
+                className={`rounded-tl-md rounded-bl-md w-16 md:w-20 sm:w-24 py-2 md:py-4 px-2 h-full sm:px-8 font-bold bg-[#101014] text-light-900 text-sm sm:text-base flex items-center justify-center group`}
```

Comment:
```suggestion
                className={`rounded-tl-[2rem] rounded-bl-[2rem] w-16 md:w-20 sm:w-24 py-2 md:py-4 px-2 h-full sm:px-8 font-bold bg-[#101014] text-light-900 text-sm sm:text-base flex items-center justify-center group`}
```

Reply:
I have pushed the changes, I have let the icons to be in the center, imo it looks weird if I made the icon stick to top.